tests/behat/features/03-searching-object.feature

```diff
@@ -74,13 +74,6 @@ Feature: test object displays after search
     When I display random object from file
     Then it is possible to add to a list
 
-
-  @api @seek026 @seekNologin @regression @cci
-  Scenario: S026 Search and display material while not logged in should not show all buttons
-    Given filename 'creator.dat' is used
-    When I display random object from file
-    Then it is not possible to add to a list
```

Comment:
Consider keeping this scenario and replacing with  `Then it should be possible to add to a list`